requirements.txt

```diff
@@ -2,7 +2,7 @@
 # Also see requirements.dev.txt
 
```

Comment:
Also remove the py2 libraries below

Reply:
I added TODOs to explain why we still need them for a while.  

I think I could do an immediate follow-up PR to remove the need for the mock dep.   

We will still need flask for py3, but we could upgrade to a more recent version.  Upgrading flask might take some research, code changes, and testing.